javatests/arcs/core/host/ParticleRegistrationTest.kt

```diff
@@ -25,15 +25,27 @@ class ParticleRegistrationTest {
 
         val hostRegistry = ExplicitHostRegistry()
         val schedulerProvider = JvmSchedulerProvider(coroutineContext)
-        hostRegistry.registerHost(JvmProdHost(schedulerProvider, ::TestProdParticle.toRegistration()))
-        hostRegistry.registerHost(TestHost(schedulerProvider("foo"), ::TestHostParticle.toRegistration()))
+
+        val dynamicRegistration = TestConstructedParticle::class.toParticleIdentifier() to
```

Comment:
I would expect you should be able to do `::TestConstructedParticle.toRegistration()` if you get rid off the build function